InvenTree/report/api.py

```diff
@@ -292,7 +294,7 @@ class StockItemTestReportPrint(StockItemTestReportMixin, ReportPrintMixin, Retri
     def report_callback(self, item, report, request):
         """Callback to (optionally) save a copy of the generated report"""
 
-        if common.models.InvenTreeSetting.get_setting('REPORT_ATTACH_TEST_REPORT', cache=False):
+        if common.models.InvenTreeSetting.get_setting('REPORT_ATTACH_TEST_REPORT'):
```

Comment:
@miggland Question: why was this changed?

Reply:
Well spotted. That was not intentional, must have occurred in a merge operation. I've changed it back.